packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -56,16 +57,12 @@ function instrumentationPayload(name: string) {
   return { object: `component:${name}` };
 }
 
-function componentFor(
-  name: string,
-  owner: Owner,
-  options?: LookupOptions
-): Option<Factory<{}, {}>> {
+function componentFor(name: string, owner: Owner): Option<Factory<{}, {}>> {
```

Comment:
This looks like an unrelated-to-types change – is this a dead code path or something?

Reply:
The type was incorrect. `factoryFor` doesn't take options.

Reply:
Glad we caught it by doing this, then. Thanks! 😃